.ci/magician/github/get.go

```diff
@@ -98,6 +104,33 @@ func (gh *Client) GetPullRequestPreviousReviewers(prNumber string) ([]User, erro
 	return result, nil
 }
 
+func (gh *Client) GetPullRequestComments(prNumber string) ([]PullRequestComment, error) {
+	url := fmt.Sprintf("https://api.github.com/repos/GoogleCloudPlatform/magic-modules/issues/%s/comments", prNumber)
+
+	var comments []PullRequestComment
+	err := utils.RequestCall(url, "GET", gh.token, &comments, nil)
+	if err != nil {
+		return nil, err
+	}
+	return comments, nil
+}
+
+func (gh *Client) GetPullRequestCommentsByUser(prNumber, login string) ([]PullRequestComment, error) {
```

Comment:
Are we expecting to reuse this? What would you think about just keeping it to GetPullRequestComments (which will be easier to port over to the standard GH client once we get around to that conversion) and handling the rest as part of FindReviewerComment?

Reply:
Removing this altogether since reviewer comment doesn't necessarily come from a specific user.